packages/stark-build/config/build-utils.js

```diff
@@ -31,13 +31,20 @@ function readTsConfig(tsConfigPath) {
 
 function getEnvironmentFile(environment) {
 	if (typeof environment === "string") {
-		let fileName = helpers.root("src/" + getAngularCliAppConfig().environments[environment]);
+		let fileName = helpers.root("src/environments/environment.ts");
```

Comment:
Could you maybe add a description to this function? Because it is not that clear what it does...

Reply:
done